### DISCUSSION TITLE
kernels filter how?

### DISCUSSION THREAD

**Discussion by mertgoksel** [microsoft/vscode-jupyter#10130] [Questions and Answers] [answered]
I want to filter out my normal python installation as the packages required for me are installed in the virtual env one. But i dont know the json code required to put it into action. I tried adding this line to the settings which did not work. There is no pattern for this specific setting on the internet.

`"jupyter.kernels.filter": [{"path": "C:\\Python\\Python_3.10.3\\python.exe", "type": "pythonEnvironment"}]`

**Comment by DonJayamanne** [comment, marked as answer]
You can go into the command `Juptyer: Filter Kernels` (its easiest to use the GUI to add the values into the settings file rather than manually updating it)

from there select what you would like to see and unselect what you don't want to see in the list of kernels.
This filter applies per workspace, if you want this setting to apply across your machine, then copy the value `"jupyter.kernels.filter": [` from your workspace `.vscode/settings.json` into your user settings file.

**Comment by krugeremily** [comment]
@DonJayamanne apologies but I don't quite understand your answer. What exactly go you mean by "go into the command" ? I can only find an option to add filters on kernels in the json code but not in a UI menu
